tests/jax/test_device_initialization.py

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import pytest
+from infra import random_tensor
+from infra.device_connector import DeviceConnector
+
+# ---------- Fixtures ----------
+
+
+@pytest.fixture(autouse=True)
+def initialize_device():
+    """
+    Initialize available devices by registering PJRT plugin.
+
+    PJRT plugin is registered during DeviceConnector instantiation. By providing global
+    singleton instance of the connector in device_connector.py we solved the problem of
+    automatic plugin registration.
+
+    We can simulate it here by calling the constructor directly.
+
+    NOTE we cannot manually do `xla_bridge.register_plugin` here because it will
+    report `PJRT_Api already exists for device type tt`.
+    """
+    DeviceConnector()
```

Comment:
Wouldn't `from infra.device_connector import device_connector` force this initialization to happen? I don't think you need this fixture